python/lsst/meas/algorithms/convertReferenceCatalog.py

```diff
@@ -262,6 +421,18 @@ def assertAllOrNone(*names):
             raise ValueError(
                 '"epoch_name" must be specified if "pm_ra/dec_name" or "parallax_name" are specified')
 
+        # Need all the error field names set if we are including covariances.
```

Comment:
Why did you remove the check that if `full_position_information` is set, the manager must be overridden?

Reply:
I requested it [in a previous comment](https://github.com/lsst/meas_algorithms/pull/299#discussion_r1024623834), because using type checks to see if something is implemented is an anti-pattern.

Reply:
I understand that, but doing the check in validation makes the failure explicit ("you need to set a config to get this functionality"), whereas waiting for the not implemented method to raise will cause a cascade of exceptions from multiprocessing.